cosmos/txbuilder.go

```diff
@@ -0,0 +1,48 @@
+package cosmos
+
+import (
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+	authutils "github.com/cosmos/cosmos-sdk/x/auth/client/utils"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/tendermint/tendermint/types"
+)
+
+// TxBuilder implements a transaction context created in SDK modules.
+type TxBuilder struct {
+	authtypes.TxBuilder
+}
+
+// NewTxBuilder returns a new initialized TxBuilder.
+func NewTxBuilder(cdc *codec.Codec, accNumber, accSeq uint64, kb *Keybase, chainID string) TxBuilder {
```

Comment:
```suggestion
func NewTxBuilder(cdc *codec.Codec, accNumber, accSeq uint64, kb *keys.Keybase, chainID string) TxBuilder {
```

We don't need to pass our struct, as TxBuilder needs the interface.